src/views/task-list-page/task-list-loader/task-list-loader.tsx

```diff
@@ -20,8 +24,10 @@ export default function TaskListLoader(props: Props) {
 
   return (
     <PageSection>
-      <div>Placeholder for Task List table</div>
-      <div>{JSON.stringify(taskList)}</div>
+      <styled.TaskListContainer>
+        <TaskListLabel taskList={taskList} isHighlighted={true} />
```

Comment:
Since the search is only for workers, i would expect the page title to be tasklist name. So it may need to moved to the top.

Reply:
Also can you explain isHighlighted usage

Reply:
If we are showing multiple tasklists on one page, the selected one will have a bolder title